banzai_floyds/orders.py

```diff
@@ -404,7 +404,9 @@ class OrderSolver(Stage):
     ORDER_HEIGHT = 93
     CENTER_CUT_WIDTH = 31
     POLYNOMIAL_ORDER = 3
-    ORDER_REGIONS = [(0, 1700), (630, 1975)]
+
+    ORDER_REGIONS = {'ogg': [(0, 1550), (500, 1835)],
```

Comment:
I have added site specific order regions as a starting guess since the order placement on the chip is different at each site. Previously this manifested in weird ways when you would start dealing with data off one of the slits for one of the sites because the defaults had been set for the other.

Reply:
This could also be runtime config in the helm chart to make that process easier?

Reply:
You certainly aren't wrong that this is a stumbling point. We have only changed cameras in floyds once in the last decade (literally) so I'm hesitant to spend the effort now to come up with a general fix for this problem. I'll put it on the longer term nice to have features list.